app/views/shared/_nav_bar.html.erb

```diff
@@ -0,0 +1,109 @@
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <% if current_admin.accessible_facilities(:manage).any? || current_admin.power_user? %>
+                            <h6 class="dropdown-header">
+                                Settings
+                            </h6>
+                            <%= link_to (admin_facilities_path), class: "dropdown-item #{active_controller?("admin/facilities")}" do %>
+                                <i class="fas fa-hospital"></i> Facilities
+                            <% end %>
+                            <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admin_organizations_path), class: "dropdown-item #{active_controller?("admin/organizations")}", id: "nav-organizations-link" do %>
+                                    <i class="fas fa-network-wired"></i> Organizations
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admin_protocols_path), class: "dropdown-item #{active_controller?("admin/protocols")}" do %>
+                                    <i class="fas fa-prescription-bottle-alt"></i> Protocol medications
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.accessible_facilities(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admins_path), class: "dropdown-item #{active_controller?("admins")}" do %>
+                                    <i class="fas fa-user-friends"></i> Dashboard admins
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.accessible_users(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admin_users_path), class: "dropdown-item #{active_controller?("admin/users")}", id: "mobile-app-users" do %>
+                                    <i class="fas fa-user-md"></i> Mobile app users
+                                <% end %> 
+                            <% end %>
+                            <% if current_admin.power_user? %>
+                                <%= link_to ("/sidekiq"), class: "dropdown-item #{active_controller?("sidekiq")}" do %>
+                                    <i class="fas fa-running"></i> Sidekiq
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.power_user? %>
+                                <%= link_to ("/flipper"), class: "dropdown-item #{active_controller?("flipper")}" do %>
+                                    <i class="fas fa-circle-notch"></i> Flipper
```

Comment:
Could I suggest a flag-like icon for this if available? (because feature flags)

Reply:
Great idea, updating to use a flag icon 😄